docs/how-to/index.rst

```diff
@@ -37,5 +37,13 @@ Configure Real-time Ubuntu Core:
 For tuning an existing Ubuntu Core system, refer to :ref:`ubuntu-core-kernel-cmdline`.
 
 
+Download Real-time Ubuntu Kernel Source Code:
```

Comment:
This is mostly redundant:
![image](https://github.com/user-attachments/assets/ca19ec03-42b8-4d7b-8baa-006452bf18bb)

```suggestion
Access source code:
```